### DISCUSSION TITLE
How do I paste from Windows to Linux via remote?

### DISCUSSION THREAD

**Discussion by ValentinVoigt** [neovide/neovide#1289] [Q&A]
Hi! I'm using neovide on Windows (natively, not WSL). I run neovide with `--remote-tcp` to connect to my development VM, which runs Linux. This works great.

But how can I paste from my Windows system clipboard? I used to use Shift+Insert when running neovim from a terminal via ssh. But with neovide that just inserts `<S-Insert>`. Luckily I found out that middle mouse button inserts correctly. But it shouldn't come as a surprise that I don't particularly like using my mouse when writing code. 😁 

I found issue #113, but I'm unsure if that is related to my problem. I also found [this unanswered StackOverflow question](https://stackoverflow.com/questions/71749548/neovide-how-to-paste-with-shiftinsert), which sounds similar, but has a completely different setup.

Any help would be greatly appreciated!

(Copying from neovide to the system clipboard, only via special commands / registers, not with every yank, would be nice, too, but is optional to me.)

**Comment by binary4cat** [comment]
I have the same question, hope someone can help😅
